buildtest/cli/report.py

```diff
@@ -954,3 +958,17 @@ def print_report_summary_output(
     console.print(table)
     pass_results.print_report(title="PASS Tests", color=color)
     fail_results.print_report(title="FAIL Tests", color=color)
+
+
+def report_detailed(report, configuration):
+    """This method will print detailed summary test results which can be retrieved via ``buildtest report --detailed`` command
```

Comment:
Some minor changes to docstring say the following

"This method will print a detailed test report which can be retrieved via ``buildtest report --detailed`` command"

Reply:
Fixed

Reply:
As discussed this method needs to be removed